From bb09ec47e70e2f00d0370791c1d96e1dda4ad952 Mon Sep 17 00:00:00 2001
From: Jacob Stiffler <j-stiffler@ti.com>
Date: Wed, 9 Sep 2015 13:57:29 -0400
Subject: [PATCH 1/9] dts/am57xx-beagle-x15: Increase dsp2 CMA region

* The extra space is needed for the opencl monitor firmware to add
  GateMP support.

Signed-off-by: Jacob Stiffler <j-stiffler@ti.com>
---
 arch/arm/boot/dts/am57xx-beagle-x15.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/am57xx-beagle-x15.dts b/arch/arm/boot/dts/am57xx-beagle-x15.dts
index 934a1e7..704b312 100644
--- a/arch/arm/boot/dts/am57xx-beagle-x15.dts
+++ b/arch/arm/boot/dts/am57xx-beagle-x15.dts
@@ -58,7 +58,7 @@
 
 		dsp2_cma_pool: dsp2_cma@9f000000 {
 			compatible = "shared-dma-pool";
-			reg = <0x9f000000 0x800000>;
+			reg = <0x9f000000 0x1000000>;
 			reusable;
 			status = "okay";
 		};
-- 
1.9.1

